src/wallet/rpcwallet.cpp

```diff
@@ -3081,6 +3084,18 @@ UniValue listunspent(const JSONRPCRequest& request)
 
         if (options.exists("maximumCount"))
             nMaximumCount = options["maximumCount"].get_int64();
+
+        if (options.exists("coinType")) {
+
+            int64_t nCoinType = options["coinType"].get_int64();
+
+            if (nCoinType > static_cast<int64_t>(CoinType::MAX_COIN_TYPE)) {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid coinType selected. Available range: %d - %d", static_cast<int64_t>(CoinType::ALL_COINS), static_cast<int64_t>(CoinType::MAX_COIN_TYPE)));
+            }
+
+            coinControl.nCoinType = static_cast<CoinType>(nCoinType);
+        }
+
```

Comment:
nit: I would drop most if not all of these empty lines, but that's mostly preference

Reply:
can you live with half of them like in ef2d027? 😅